tests/coalaJSONTest.py

```diff
@@ -53,7 +54,9 @@ def test_fail_acquire_settings(self):
 
     def test_show_all_bears(self):
         with bear_test_module():
-            retval, output = execute_coala(coala.main, 'coala', '--json', '-B')
+            retval, output = execute_coala(coala.main,
+                                           'coala', '--json', '-B',
+                                           stdout_only=True)
```

Comment:
It is related because the `default` section deprecation warnings were showing in the output and that was making the JSON invalid.